src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax;
+
+public enum Colors {
+    black,
+    white,
+    red,
+    blue
+}
```

Comment:
enums write in UPPER_CASE
```suggestion
public enum Colors {
    BLACK,
    WHITE,
    RED,
    BLUE
}
```